src/components/Typography/h2.js

```diff
@@ -3,10 +3,16 @@ import remcalc from 'remcalc'
 import styled from 'styled-components'
```

Comment:
couldn't we just 

```
<H1 as="h2">{children}</H1>
``` 
?

Reply:
It's the same thing but I find this one buildable upon if we need to add something

Reply:
I updated the h2